packages/open-api-gateway/src/construct/open-api-gateway-lambda-api.ts

```diff
@@ -331,6 +346,16 @@ export class OpenApiGatewayLambdaApi extends Construct {
       });
     });
 
+    // Create and associate the web acl
+    const acl = new OpenApiGatewayWebAcl(this, `${id}-Acl`, {
```

Comment:
Shouldn't we should only create this if `!props.webAclOptions.disable`?

I know you have it within the construct itself but it doesn't feel like the right place to me.

Reply:
Sure I can move it out :)